### ISSUE TITLE
DoES Liverpool photos on Google maps are of Hanover Street

### ISSUE THREAD

**Issue by RiverMersey** [DoESLiverpool/somebody-should#1358]
While much of the design and equipment are mostly the same, I've only just noticed that most of the internal photos and 360s of DoES is still of the Hanover Street Building.

Google has a feature of suggesting edits but I can't quite figure out how to suggest edits of old internal photos for present-day ones.

Not vitally important, but it seems that the Hanover Street photos meta-tags also causes Google to display a mini-map of the location of the Hanover Street Building.
![does liverpool old pic](https://user-images.githubusercontent.com/54508654/71331354-b6dc5000-2529-11ea-9779-77a5f14fa96a.gif)

**Comment by MatthewCroughan** [comment]
I've made this panorama with the aforementioned gimbal and my phone as a test:

![IMG_20191223_151143 - IMG_20191223_151253_1_blended_fused_compressed](https://user-images.githubusercontent.com/26458780/71374556-15014580-25b3-11ea-9c37-8d3c38de0e7e.jpg)

**Comment by RiverMersey** [comment]
Yes, yes yes! Wow, you have had a very busy and productive day! 👍😁

**Comment by MatthewCroughan** [comment]
The panorama was generated with Hugin, thanks much to @Thingomy for suggesting this almost a year ago. There's too much parallax to get a *perfect* panorama since I used the camera in horizontal orientation rather than vertical, but when these are done next time they will be held vertically to reduce parallax.

**Comment by johnmckerrell** [comment]
For the record, I tried to edit this today and it looks like we don't actually have access to it, it's currently claimed by someone with an email address beginning with `cl***@....`. I've requested access so if it's someone here can you transfer it please.

**Comment by johnmckerrell** [comment]
I now have access, I've had to flag the old photos as it won't let you just delete them. I've focussed on the 360s as the others are less clear where they're taken and show people or equipment (and it's a PITA to flag things). I've also uploaded a load of more recent photos. I've also set a "short name" so that we can be found at https://g.page/DoESLiverpool?gm

(Looks like it was owned by Paul Freeman, I've added others as owners now alongside me)
(Seems like we probably shouldn't have edited the address but added a new "location", that way we'd have had a fresh set of photos)

**Comment by RiverMersey** [comment]
Similarly, some or the info stored on this GitHub about DoES Liverpool is also outdated - mainly by referring to the Hanover Street Building.

For example, DoESLiverpool/somewhere-safe has an entry about disability parking and access.

Perhaps this should be raised as a separate "somebody should" but it's kind of related to the theme of updating the Digital DoES Liverpool identities for the new building and location.

**Comment by johnmckerrell** [comment]
I occasionally think about doing what we "should" have done originally, creating a new "location" in the right place (and deleting or "closing" the previous one). The thing that puts me off is if we would lose out by having such a "new" place but I wonder if now is a good time to do it given that people aren't really /going/ to places. Does anyone have experience with businesses on Google that could say whether or not this is a good idea? @willcrobinson maybe?

**Comment by willcrobinson** [comment]
Sorry, I've not dealt with a business moving location before! The main worry would be that we might lose the 40 reviews that DoES has built up over the years. Google apparently offer really good phone support when it comes to MyBusiness (you just fill out the form here and they call you back right away https://support.google.com/business/gethelp?visit_id=637478094788155996-2441418680&rd=1) It might be worth asking them about removing the offending photos of Gostins?

**Comment by johnmckerrell** [comment]
Ah yes, I knew there was something I was worried about losing! That sounds like good advice, I wonder if @DoESsean might do that?

**Comment by DoESsean** [comment]
Right, I've been into the Business dashboard, where you're meant to be able to delete photos, and can't figure out how to do it. The instructions are for some other iteration of the current site entirely.

Anyway, I've raised issued for all the photos from Gostins that focus on the building, rather than just a shot of the equipment or something, saying that they're of our old location and no longer accurately represent our office, so we'll see what comes back. 

The call back service is currently unavailable.

**Comment by johnmckerrell** [comment]
FWIW I also flagged the photos a year ago but nothing happened.

**Comment by DoESsean** [comment]
Joy.

Let's see if it works this time.

**Comment by willcrobinson** [comment]
Not sure if it'll make any difference but I've flagged all of the Gostins photos too.

**Comment by DoESsean** [comment]
Right, I've just downloaded the My Business app, as it appears you need to do it through that.

Problem is, I can't delete any photos from our listing that haven't been uploaded by us, I can only flag them. Looks like we're at the mercy of Google...

**Comment by johnmckerrell** [comment]
I wonder if we create a new "location" and remove the old one if we would lose the reviews then, and if we would (and perhaps giving some time to try again to get in touch with their support) if we should just accept that.
